packages/ember/index.js

```diff
@@ -427,6 +428,7 @@ Ember._concat = concat;
 Ember._get = get;
 Ember._on = on;
 Ember._fn = fn;
+Ember._uniqueId = uniqueId;
```

Comment:
(For us as maintainers, not for you as a contributor!)

I will put this on the agenda for tomorrow's Framework team meeting; as a matter of policy I think we should *avoid* adding things to the `Ember` namespace.

Reply:
Confirmed in today’s discussion: we will *not* add this to the `Ember` namespace, so we can remove this addition entirely!

```suggestion
```